src/api/models/iuf/iuf-manifest-schema.yaml

```diff
@@ -437,3 +462,21 @@ properties:
               description: >
                 The key of the object to be created in S3.
 
+$defs:
+  MD5Sum:
+    type: string
+    pattern: "^[0-9a-fA-F]+$"
```

Comment:
are they always hex-encoded strings here?

Reply:
Yes, `python-ims-helper` is using [hashlib.hash.hexdigest()](https://docs.python.org/3/library/hashlib.html?highlight=hexdigest#hashlib.hash.hexdigest) which returns a hexadecimal string. On second glance I think I need to remove the uppercase `[A-F]` character set, since MD5 validation is just a string comparison.